src/main/java/blue/endless/jankson/api/document/ObjectElement.java

```diff
@@ -167,7 +167,22 @@ public String toString() {
 		w.flush();
 		return w.toString();
 	}
-	
+
+	/**
+	 * Gets the KeyValuePairElement entry if it exists in this object.
+	 * If no element exists with the specified key then null will be returned.
```

Comment:
We got out of sync with the documentation

Reply:
Oops, I'm tired.

Reply:
Fixed